src/types/proof_of_space.py

```diff
@@ -47,11 +46,12 @@ def verify_and_get_quality_string(
         if not ProofOfSpace.passes_plot_filter(constants, plot_id, original_challenge_hash, signage_point):
             return None
 
-        quality_str = v.validate_proof(plot_id, self.size, self.challenge, bytes(self.proof))
+        return self.get_quality_string(plot_id)
 
+    def get_quality_string(self, plot_id):
```

Comment:
Can you specify types in the method arguments and return types? Please do this for all methods